lstchain/io/event_selection.py

```diff
@@ -19,7 +19,7 @@ class EventSelector(Component):
     taken as inputs and filter_events() is used on a table of events
     called in with the Component.
 
-    For event_type, we choose the sub-array trigger of 32, which is for 
+    For event_type, we choose the sub-array trigger of 32, which is for
     gamma-ray event candidate, as per the latest CTA R1 Event Data Model.
```

Comment:
Not gamma-ray candidate (except "in a very loose way"), but shower candidate or "cosmics" trigger.
I did not understand why the 33 - this is just like an upper cut? Not needed, because filters are min <=  x <= max

Also, you can import ctapipe.instrument.EventType  and use EventType.SUBARRAY.value instead of 32

Reply:
No, you won't need to add 1, since filter is using `lower <= value <= upper`, so having the filter just be `[EventType.SUBARRAY.value, EventType.SUBARRAY.value]` is fine:

https://github.com/cta-observatory/cta-lstchain/blob/74e35aebfe6a6f94ba2083e5cd342355668f9565/lstchain/reco/utils.py#L529